ml/ml.cc

```diff
@@ -138,6 +138,50 @@ char *ml_get_anomaly_event_info(RRDHOST *RH, const char *AnomalyDetectorName,
     return strdup(Json.dump(4, '\t').c_str());
 }
 
+char *ml_get_anomaly_rate_info(RRDHOST *RH, time_t After, time_t Before) {
+    if (!RH || !RH->ml_host) {
+        error("No host");
+        return nullptr;
+    }
+
+    Host *H = static_cast<Host *>(RH->ml_host);
+    std::vector<std::pair<std::string, double>> DimAndAnomalyRate;
+    if(Before > After) {
```

Comment:
I think it's better to assume that this function will not be called with non-sensical values from the backend. It will simplify the logic and make things easier to reason about.

Reply:
I'd rather leave it as is, only adds to safety, even if backend is supposed to comply.